scratch/copy-mk.py

```diff
@@ -114,7 +115,8 @@ async def async_main(args) -> int:
     # This returns a string representation of a list of tuples, with each item
     # in the format (input-label, input-index, LRU host, index-on-host).
     input_labelling_str = await corr2_client.sensor_value("input-labelling", str)
-    input_labels_list = [labels_tuple.split(",")[0][1:-1] for labels_tuple in input_labelling_str[1:-1].split("(")[1:]]
+    input_labelling: list[tuple[str,]] = ast.literal_eval(input_labelling_str)
```

Comment:
```suggestion
    input_labelling: list[tuple[str, ...]] = ast.literal_eval(input_labelling_str)
```
(what you wrote is a list of tuples with each tuple containing exactly one str)

Reply:
Oh gosh, apologies. I started writing that type hint then got sidetracked when we started discussing the input labels.

Fixing now. For the record, it's in the format (string, int, string, int).
![image](https://github.com/user-attachments/assets/cb2a47ef-6f3a-459e-a9e0-7868cc889ba9)

Reply:
Oh good point, it's not all strings. So `list[tuple[str, int, str, int]]` then.